schedule/yast/minimal+base/minimal+base@yast-pvm.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: minimal+base@yast-pvm
+description: >
+  Select a minimal textmode installation by starting with the default and unselecting all patterns
+  except for "base" and "minimal". Not to be confused with the new system role "minimal" introduced with SLE15.
+vars:
+  DEPENDENCY_RESOLVER_FLAG: '1'
+  DESKTOP: textmode
```

Comment:
this one we can pull out as it is set at the level of machine.

Reply:
I added one conditional for 'reconnect_mgmt_console'.